src/Link.ts

```diff
@@ -23,11 +23,12 @@ const getProperties = (router: Router<any>, properties: any): LinkProperties =>
 			onClick(event);
 		}
 
-		if (!event.defaultPrevented && event.button === 0 && !properties.target) {
+		if (!event.defaultPrevented && event.button === 0 && !properties.target && typeof href === 'string') {
```

Comment:
I think we want to prevent default even if there is no `href` and just not call the `router.setPath` when `href` undefined.